Source/VCGeneration/Checker.cs

```diff
@@ -387,297 +381,6 @@ public ProverInterface.Outcome ReadOutcome()
   // -----------------------------------------------------------------------------------------------
   // -----------------------------------------------------------------------------------------------
 
-  public abstract class ProverInterface
```

Comment:
Apart from being moved this class only has had the method `SetRandomSeed` removed.